src/components/trip_view/TripView.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import TripCalContainer from './cal/TripCalContainer';
+import { CalViewOptions } from './cal/CalViewOptions';
+
+/**
+ * Placeholder component for the view that allows a user to view
+ * all the events contained within a trip. It currently contains
+ * two pieces: the calendar view itself (which will eventually
+ * have an agenda view and a to-do view) as well as an options
+ * component which can be placed within a sidebar down the road
+ * (the options are completely separate from the calendar
+ * component).
+ */
+export class TripView extends Component {
+  render() {
+    return (
+      <div>
+        <CalViewOptions history={this.props.history} />
```

Comment:
Woah, you actually have to manually deal with the history? I take it this was part of pulling the menu apart? Everything I've read said to not deal with history and let the browser router take care of it, but I believe you that it is necessary here (I am assuming its part of the path you created for different views?) Anyways, nice job figuring that one out...

Reply:
Yes, I had a really hard time figuring out how to redirect to a new view without using the history object! If there's a better way, yeah, I'd definitely love to do something more elegant.

Reply:
Based off of this conversation, I assume you guys have encountered some issue related to history already? If so, what were those symptoms? This stuff is new to me.

Reply:
I haven't experienced any problems. However, it's not elegant how you have to pass a history object from parent to child at every stage. It's probably poor form to do this, as well.